forum.feature

```diff
@@ -34,22 +34,21 @@ Feature: Plagiarism plugin works with a Moodle forum
       | turnitinenabled               | 1                              |
       | accessoptions[accessstudents] | 1                              |
     And I follow "Test forum"
-    And I click on "Add a new discussion topic" "button"
-    And I click on "I accept the Turnitin EULA" "button"
+    And I follow "Add a new discussion topic"
     And I set the following fields to these values:
       | Subject | Forum post 1                                                                                                                |
       | Message | This is the body of the forum post that will be submitted to Turnitin. It will be sent to Turnitin for Originality Checking |
     And I press "Post to forum"
 
-  @javascript
+  @javascript @_file_upload
   Scenario: Add a post to a discussion with a file attached and retrieve the originality score
     Given I log out
     # Student creates a forum discussion and replies to original post.
     And I log in as "student1"
     And I am on "Course 1" course homepage
     And I follow "Test forum"
-    And I click on "Add a new discussion topic" "button"
-    And I click on "I accept the Turnitin EULA" "button"
+    And I follow "Add a new discussion topic"
+    #And I click on "I accept the Turnitin EULA" "button"
```

Comment:
I don't think this is meant to be commented out.

Reply:
I think by mistake I have left it there. I have double checked now there is no eula present at this page

Reply:
Odd that there's no EULA there. I thought the step would only accept it if the EULA is shown but I checked it and don't see logic for that. Does the test pass if you run it on its own?

Reply:
Yeah it worked fine for me. I ran it with keeping this step and it failed there but when I removed the step it just worked fine.